src/Source/CsvSourceIterator.php

```diff
@@ -14,17 +14,17 @@
 /**
  * Read data from a csv file.
  *
- * @author Vincent Touzet <vincent.touzet@gmail.com>
```

Comment:
This should not be changed.

At least, you can add your author tag below, but not changing the current one.